src/components/PanelsList.js

```diff
@@ -93,7 +84,7 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     width: "100%",
-    height: "100%"
+    height: "100%",
```

Comment:
то схоже випадково додалось
```suggestion
    height: "100%"
```